ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.tsx

```diff
@@ -57,6 +57,8 @@ const colors = [red, green, yellow, blue, magenta, cyan];
 const reset = '\u001b[0m';
 const whiteOnYellow = '\u001b[1m\u001b[43;1m\u001b[37m';
 
+const POD_BACKGROUND_COLORS = ['rgba(230, 243, 255, 0.7)', 'rgba(255, 243, 230, 0.7)', 'rgba(230, 255, 230, 0.7)', 'rgba(255, 230, 230, 0.7)', 'rgba(243, 230, 255, 0.7)'];
```

Comment:
I am wondering if there have to be hard coded or can be gathered from an environment variable ...

Reply:
Also, need to adjust for the dark mode.

Reply:
you can utilise argo-ui color variables with altreanates to light blue and White in light theme, and light gray and white in dark theme.